config/schema_mysql.sql

```diff
@@ -24,4 +24,12 @@ end_time TIMESTAMP,
 destination_type VARCHAR(99) NOT NULL,
 destination_url VARCHAR(99),
 status CLOB,
-PRIMARY KEY (MIGRATION_ID));
\ No newline at end of file
+PRIMARY KEY (MIGRATION_ID));
+
+-- create table box_auth_user
+create table box_auth_user (
```

Comment:
I don't know whether these files of SQL queries are used to migrate schema changes to production.  If there's a chance that they could be and some of the tables may exist when the queries are run, it would be a good idea to change the `create table` query to:

```sql
create table if not exists box_auth_user (
```

etc.